src/components/menu/components/item-menu.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {connect} from "react-redux";
+
+import '../menu.css';
+import '../../app/App.css'
+
+const ItemMenu = ({title, operation, lang}) => {
+	const fontSize = {
+		fontSize: (lang === 'en') ? '2rem' : '1.5rem',
+	};
+	return (
+		<button className="btn btn-success btn-lg btnShadow" style={fontSize}

```

Comment:
to avoid setting styles inline, it is better to set appropriate class (depending on lang), and set in css